GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -15,6 +15,8 @@
 using Microsoft;
 using Microsoft.WindowsAPICodePack.Dialogs;
 using ResourceManager;
+using TaskDialog = Microsoft.WindowsAPICodePack.Dialogs.TaskDialog;
+using TaskDialogStartupLocation = Microsoft.WindowsAPICodePack.Dialogs.TaskDialogStartupLocation;
```

Comment:
💡 There are some other files that would benefit from this approach. And I was literally just mentioning this to a team member in context of https://github.com/dotnet/roslyn/issues/38095. 😄

Reply:
These will be replaced with [`System.Windows.Forms.TaskDialog`](https://docs.microsoft.com/dotnet/api/system.windows.forms.taskdialog)